templates/plugin-bitbucket.mustache

```diff
@@ -0,0 +1,45 @@
+pipelines:
+  default:
+    - step:
+        image: tfirdaus/wp-docklines:php5.6-fpm-alpine
```

Comment:
It'd be good to use something more neutral here (and passim) I think rather than your personal stuff! Is there a way to do that??

Reply:
Hi, @gitlost 
Thank you for reviewing the pull request.

Yes, that's possible. However, in the case of Bitbucket Pipelines, I'm afraid doing so might harm users sitting on the Free plan as Bitbucket *only* allocates 50 minutes of build per month [on the Free plan](https://bitbucket.org/product/pricing). Bitbucket suggests creating our own Docker images that are ready to go as setting up dependencies and tools during the build will be time-consuming.

Here is what that says in one of their articles [Tips for scripting tasks with Bitbucket Pipelines
](https://www.atlassian.com/continuous-delivery/tips-for-scripting-tasks-with-Bitbucket-Pipelines)

> Installing the dependencies necessary for your pipeline to run can be time-consuming. You could save a lot of running time by creating your own Docker image with the basic tools and packages required to build and test your application.

Hence the reason for those _personal_ images 🙂 

I'm open to suggestion; if including personal images is not allowed I'm more than happy to change it to a neutral image.

Reply:
Hi @tfirdaus !

> Bitbucket suggests creating our own Docker images that are ready to go as setting up dependencies and tools during the build will be time-consuming.

Ah I see, ta. (Forgive my ignorance, I know next to nothing about Docker also and only just noticed your images here https://hub.docker.com/r/tfirdaus/wp-docklines/)

So perhaps the way to go in the future might be to register `wp-cli` or similar on Docker Hub and upload the images there. But in the meantime we could use your (*oh so personal!*) account.

Thanks for the feedback, would like to get input from @schlessera on it - though we're about to release WP-CLI 1.5.0 on Tuesday so may not be in touch immediately...

Reply:
I'm happy to transfer the Docker image ownership to WP-CLI official if that's necessary 🙂as the image might also be useful for [GitlabCI](https://github.com/wp-cli/scaffold-command/blob/master/templates/plugin-gitlab.mustache) or [CircleCI](https://github.com/wp-cli/scaffold-command/blob/master/templates/plugin-circle.mustache) _scaffold command_.

Reply:
@tfirdaus I understand the reason why you added the (personal) image to the script. However, I don't want to include such a custom image in this script for the following reasons:

* Providing an official WP-CLI image means that the WP-CLI maintainers need to indefinitely maintain this image. I consider this is out of scope for the WP-CLI project.
* Providing your personal image is a potential security/trust issue. We would basically endorse an unknown black box image. I don't feel comfortable doing this, not because I don't trust you as a person, but because I think that this is just not a good idea in principle. We have no control over that image, so must assume it cannot be trusted.
* The scaffolding tools are meant to provide a starting point. Having a basic BB Pipelines setup that has a limitation on the free plan is perfectly okay. It still helps people to get started quickly, and they are meant to do adjustments to fit their needs either way.

Given the above, would you be willing to change the PR so that it doesn't include a pre-built image, but rather compiles a minimum set of dependencies on the fly?

Reply:
Understood and that totally makes sense. I'll rewrite the Pipelines config to the minimum :) Thank you for your feedback @schlessera